jira_agile_metrics/calculators/cycletime.py

```diff
@@ -95,16 +95,14 @@ def calculate_cycle_times(
         now = datetime.datetime.utcnow()
 
     cycle_names = [s['name'] for s in cycle]
-    accepted_steps = set(s['name'] for s in cycle if s['type'] == StatusTypes.accepted)
-    completed_steps = set(s['name'] for s in cycle if s['type'] == StatusTypes.complete)
```

Comment:
I kind of see why this makes sense. I think originally the idea was that we'd use the `StatusTypes` enum to mark columns as representing backlog, committed, active, done, etc. states. But in reality you're still reliant on applying the analysis to a sequential set of columns from start to finish.

Not really for this PR but I wonder if we should just get rid of the `type` option against cycles altogether, i.e. is it really doing anything?

Reply:
As described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 I think that cycle time calculations require `Committed column` and `Done column` only.
Additional attributes would only be required if we wanted to ignore intermediate workflow steps (e.g. waiting columns) in cycle time calculation.